automl/automlVisionDataset.js

```diff
@@ -0,0 +1,400 @@
+/**
+ * Copyright 2018, Google, LLC.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*eslint no-console: 0*/

```

Comment:
I don't be around to read the answer, but... just want to call this out – @happyhuman check with @CallistoCF on this `eslint` line to understand why it's here – I have zero knowledge of what this does & why :)

Reply:
I just removed this, as eslint.ymc in samples directory doesn't have this rule, this will be in newest push!